udp_server.go

```diff
@@ -257,9 +257,27 @@ func udpServer() {
 			log.Printf("received: %s from %s\n", sdata, remote)
 		}
 
+		// Check if the message has a key named "type" and rename it to "read_type"
+		if val, ok := packet["type"]; ok {
+			packet["read_type"] = val
+			delete(packet, "type")
+		}
+
 		// send data to Stomp endpoint
 		if Config.Endpoint != "" && stompConn != nil {
-			sendDataToStomp(data)
+			newData, err := json.Marshal(packet)
+			if err != nil {
+				log.Printf("unable to marshal UDP packet into JSON, error %v\n", err)
```

Comment:
This message may be useless without knowing exact packet, at the same time packet can be large and you don't want to blindly dump it to the log. I suggest to check size of the packet and dump first N bytes to the log (you'll need to convert them to string, like `string(packet[:100])` otherwise you'll get list of ints.

Reply:
Thank you. Depending on the current messages I have chosen a bit higher N, but otherwise made the required changes.